components/Vote/ElectionCandidacy.js

```diff
@@ -145,7 +145,7 @@ class ElectionCandidacy extends React.Component {
   constructor (props) {
     super(props)
     this.state = {
-      isEditing: props.router.query.hasOwnProperty('edit') || false,
+      isEditing: props.router.query['edit'] || false,
```

Comment:
```suggestion
      isEditing: !!props.router.query.edit || false,
```
should probably be cast to boolean